loader/src/ui/mods/GeodeStyle.cpp

```diff
@@ -217,19 +222,18 @@ std::string geodeTagName(std::string_view tag) {
     return readable;
 }
 
-ListBorders* createGeodeListBorders(CCSize const& size) {
+ListBorders* createGeodeListBorders(CCSize const& size, bool forceDisableTheme) {
     auto ret = ListBorders::create();
-    if (isGeodeTheme()) {
+    const bool geodeTheme = isGeodeTheme(forceDisableTheme);
+    if (geodeTheme) {
         ret->setSpriteFrames("geode-list-top.png"_spr, "geode-list-side.png"_spr, 2);
+        ret->setContentSize(size);
+    } else {
+        ret->setContentSize(size + ccp(5, 5));

```

Comment:
To explain this real quick, blame the fact that 90% of this asset is padding

![GJ_commentTop_001](https://github.com/user-attachments/assets/188e96f8-4896-4b25-9e57-b2dac4174686)

Reply:
We love hardcoded values for dealing with weird textures